ModTek/Logging/BetterLogSettings.cs

```diff
@@ -0,0 +1,21 @@
+using HBS.Logging;
+
+namespace ModTek.Logging
+{
+    public class BetterLogSettings
+    {
+        public bool Enabled = false;
+        public LogLevel Level = LogLevel.Debug;
```

Comment:
This debug level refers to the loggers log level in case of a Mods Logger, and to the file appenders loggers on case of the cleaned log

Reply:
Maybe make a copy and move it to the same location as stacktrace enabled setting

Reply:
just have to hope people dont get confused